apps/condo/domains/banking/access/BankCategory.js

```diff
@@ -0,0 +1,32 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema banking.BankCategory 'name:Text;'`
+ */
+
+const { throwAuthenticationError } = require('@condo/keystone/apolloErrorFormatter')
+
+async function canReadBankCategories ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin) return {}
```

Comment:
What about supports?

Reply:
It does not needs to be explicitly mentioned, since we are not suppose to make restrictions on read access on categories.